src/lib/operators/aggregate_sort.cpp

```diff
@@ -392,21 +394,21 @@ std::shared_ptr<const Table> AggregateSort::_on_execute() {
    *               So in total, group_boundaries will contain one element less than there are groups.
    */
   std::set<RowID> group_boundaries;
-  const auto& chunks = sorted_table->chunks();
+  const auto chunk_count = sorted_table->chunk_count();
   for (const auto& column_id : _groupby_column_ids) {
     auto data_type = input_table->column_data_type(column_id);
     resolve_data_type(data_type, [&](auto type) {
       using ColumnDataType = typename decltype(type)::type;
       std::optional<ColumnDataType> previous_value;
-      ChunkID chunk_id{0};
+      ChunkID chunk_id_outer{0};
```

Comment:
What meaning does "outer/inner" have when it comes to an aggregate? Can we find a more descriptive name that does not reference the position in the code?

Reply:
Ok, I went through the code: A single variable `chunk_id` is sufficient. I was quite irritated by this, but `chunk_id_outer` has no function. Therefore, the naming questing is solved.